### PR TITLE
[HOLD for payment 2023-11-30] Manually manage tags in bumpVersion action

### PR DESCRIPTION
**Pull request by roryabraham**
### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/155197

### Tests
None yet, test by merging

### Tested On

- Github

### CODE REVIEW

File: .github/actions/bumpVersion/bumpVersion.js [hunk 1]
```diff
@@ -84,11 +94,17 @@ do {
...
             .then(({stdout}) => {
-                // NPM and native versions successfully updated - don't retry.
+                // NPM and native versions successfully updated, create a tag
                 console.log(stdout);
+                const addCommand = `git add ${
+                    [PACKAGE_JSON_PATH, PACKAGE_LOCK_PATH, BUILD_GRADLE_PATH, PLIST_PATH, PLIST_PATH_TEST].join(' ')
+                }`;
+                const commitCommand = `git commit -m "Update version to ${newVersion}`;
+                const tagCommand = `git tag ${newVersion} && git push --tags`;
+                return exec(`${[addCommand, commitCommand, tagCommand].join(' && ')}`);
```
Comment (AndrewGable) [inline .github/actions/bumpVersion/bumpVersion.js:107]: Not sure I have an opinion either way, but is there a reason for doing this in JS as opposed to in the action?

Comment (roryabraham) [reply .github/actions/bumpVersion/bumpVersion.js:107]: The only reason would be to use the new version in the commit message and tag. However, one of the things on our project roadmap is having this action output the new version, so we could just output the new version from JS then do the git stuff in the workflow.

Comment (AndrewGable) [reply .github/actions/bumpVersion/bumpVersion.js:107]: Discussed offline and I think we both agreed that moving this out of JS will make our lives and code easier in the long run

File: .github/workflows/version.yml [hunk 2]
```diff
@@ -33,10 +33,22 @@ jobs:
...
 
+            - name: Commit and tag new version
+              run: |
+                git add \
+                  ./package.json \
+                  ./package-lock.json \
+                  ./android/app/build.gradle \
+                  ./ios/ExpensifyCash/Info.plist \
+                  ./ios/ExpensifyCashTests/Info.plist
+                git cm -m "Update version to ${{ steps.bumpVersion.outputs.newVersion }}"
```
Comment (AndrewGable) [inline .github/workflows/version.yml:49]: Is `cm` a local shortcut you have installed locally?

Comment (roryabraham) [reply .github/workflows/version.yml:49]: 😅 is `git commit` even a command?

Comment (roryabraham) [reply .github/workflows/version.yml]: Good catch! I honestly forgot that `git cm` is just an alias

**Review by AndrewGable — APPROVED** [review]

**Pull request merged**